src/main/java/com/ssssogong/issuemanager/util/StringCollectionConverter.java

```diff
@@ -0,0 +1,29 @@
+package com.ssssogong.issuemanager.util;
+
+import jakarta.persistence.AttributeConverter;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+import static java.util.Collections.emptyList;
+
+/** String 컬렉션을 String으로 변환
+ *  엔티티 속성 DB에 저장할 때 converter로 활용
+ *  ex) ["ISSUE_REPORTABLE", "ISSUE_FIXABLE"] => "ISSUE_REPORTABLE;ISSUE_FIXABLE" */
+public class StringCollectionConverter implements AttributeConverter<Collection<String>, String> {
+    private static final String SPLIT_CHAR = ";";  // String 구분자
+
+    @Override
+    /** List<String> 타입의 엔티티 속성을 String 타입 DB 칼럼으로 변환 */
+    public String convertToDatabaseColumn(Collection<String> strings) {
+        return strings != null ? String.join(SPLIT_CHAR, strings.toArray(String[]::new)) : "";
```

Comment:
strings에 아무것도 들어있지 않을 수 있으니 제안 드려봅니다
```suggestion
        return !strings.isEmpty() ? String.join(SPLIT_CHAR, strings.toArray(String[]::new)) : "";
```